Sources/Logger/Loggers/UnifiedLogger/UnifiedLogger.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import OSLog
+
+/// Logger using Apple unified logging API
+/// https://developer.apple.com/documentation/os/logger
+public class UnifiedLogger: Logging {
```

Comment:
Ja bych to asi nejak prejmenoval, treba `NativeLogger`? Tohle mi prijde takovy nejednoznacny...